diff --git a/src/AniLibria.pro b/src/AniLibria.pro_new
index f627881..7c000e2 100644
--- a/src/AniLibria.pro
+++ b/src/AniLibria.pro_new
@@ -135,28 +135,28 @@ unix {
 
 #16x16
     icon_16.path = $$PREFIX/share/icons/hicolor/16x16/apps
-    icon_16.commands = mkdir -p icons/16 && cp ../anilibria.iconset/icon_16x16.png icons/16/anilibria.png
-    icon_16.files += icons/16/anilibria.png
+    icon_16.commands = mkdir -p icons/16 && cp anilibria.iconset/icon_16x16.png icons/16/anilibria.png
+    icon_16.files += ../icons/16/anilibria.png
     icon_16.CONFIG += no_check_exist
 #32x32
     icon_32.path = $$PREFIX/share/icons/hicolor/32x32/apps
-    icon_32.commands = mkdir -p icons/32 && cp ../anilibria.iconset/icon_32x32.png icons/32/anilibria.png
-    icon_32.files += icons/32/anilibria.png
+    icon_32.commands = mkdir -p icons/32 && cp anilibria.iconset/icon_32x32.png icons/32/anilibria.png
+    icon_32.files += ../icons/32/anilibria.png
     icon_32.CONFIG += no_check_exist
 #128x128
     icon_128.path = $$PREFIX/share/icons/hicolor/128x128/apps
-    icon_128.commands = mkdir -p icons/128 && cp ../anilibria.iconset/icon_128x128.png icons/128/anilibria.png
-    icon_128.files += icons/128/anilibria.png
+    icon_128.commands = mkdir -p icons/128 && cp anilibria.iconset/icon_128x128.png icons/128/anilibria.png
+    icon_128.files += ../icons/128/anilibria.png
     icon_128.CONFIG += no_check_exist
 #256x256
     icon_256.path = $$PREFIX/share/icons/hicolor/256x256/apps
-    icon_256.commands = mkdir -p icons/256 && cp ../anilibria.iconset/icon_256x256.png icons/256/anilibria.png
-    icon_256.files += icons/256/anilibria.png
+    icon_256.commands = mkdir -p icons/256 && cp anilibria.iconset/icon_256x256.png icons/256/anilibria.png
+    icon_256.files += ../icons/256/anilibria.png
     icon_256.CONFIG += no_check_exist
 #512x512
     icon_512.path = $$PREFIX/share/icons/hicolor/512x512/apps
-    icon_512.commands = mkdir -p icons/512 && cp ../anilibria.iconset/icon_512x512.png icons/512/anilibria.png
-    icon_512.files += icons/512/anilibria.png
+    icon_512.commands = mkdir -p icons/512 && cp anilibria.iconset/icon_512x512.png icons/512/anilibria.png
+    icon_512.files += ../icons/512/anilibria.png
     icon_512.CONFIG += no_check_exist
 
