diff --git a/onlineplayervideolist.cpp b/onlineplayervideolist.cpp.net
index eec0a87..3b67cf5 100644
--- a/src/Classes/ListModels/onlineplayervideolist.cpp
+++ b/src/Classes/ListModels/onlineplayervideolist.cpp
@@ -279,7 +279,7 @@ void OnlinePlayerVideoList::setVideosFromSingleTorrent(const ReleaseTorrentModel
     auto countSeries = 0;
     auto series = torrent.series();
     if (series.indexOf("-") > -1) {
-        auto parts = series.splitRef("-");
+        auto parts = series.split("-");
         auto start = parts[0].toInt();
         auto end = parts[1].toInt();
 
